dotCMS/src/main/java/com/dotmarketing/portlets/folders/business/FolderAPIImpl.java

```diff
@@ -552,15 +552,18 @@ public void save(final Folder folder, final String existingId,
 
 		Host host = APILocator.getHostAPI().find(folder.getHostId(), user, respectFrontEndPermissions);
 		Folder parentFolder = findFolderByPath(id.getParentPath(), id.getHostId(), user, respectFrontEndPermissions);
-		Permissionable parent = id.getParentPath().equals("/")?host:parentFolder;
+		Permissionable parent = "/".equals(id.getParentPath()) || "/System folder".equals(id.getParentPath()) 
```

Comment:
Would be nice to create constants for **"/System folder"** and **"/"**

Reply:
+1

Reply:
I had them on the original PR, so let me update this one as well